src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -232,6 +232,21 @@ The following options are macOS specific:\n\
 \    -Xdock:icon=<path to icon file>\n\
 \                      override default icon displayed in dock\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   Usage: java [java options...] <application> [application arguments...]\n\n\
+\Where <application> is one of:\n\
+\  <MainClass>                to execute the main method of a compiled class\n\
+\  -jar <JarFile>.jar         to execute the main class of a JAR archive\n\
+\  -m <module>[/<MainClass>]  to execute the main class of a module\n\
+\  <SourceFile>.java          to compile and execute a single-file program\n\n\
+\Where key java options include:\n\
+\  --class-path <class path>\n\
+\      a "{0}"-separated list of directories and JAR archives to search for class files\n\
+\  --module-path <module path>\n\
+\      a "{0}"-separated list of directories and JAR archives to search for modules\n\n\
+\For more details about this launcher:   java --help\n\
```

Comment:
Hello Jan, perhaps we should avoid the use of "launcher" here and instead reword it as:
```
For additional help on usage: java --help\n\
```

Reply:
Sorry, but it feels a bit incomplete - it does not say usage of what. I don't quite see the issue with the word "launcher". The `java` command is a launcher for user programs, is it not?

Reply:
> usage of what

That seems pretty self-evident to me.

"launcher" is our terminology for the java executable, but for users it is just a program that runs other stuff.

Reply:
Ok:
https://github.com/openjdk/jdk/pull/21411/commits/dc85a1679b623806a62de34c22e96793e7c0a3cd